cargo-dist/src/init.rs

```diff
@@ -84,15 +128,59 @@ pub fn do_init(cfg: &Config, args: &InitArgs) -> DistResult<()> {
         }
     };
 
+    // If we're migrating, the configuration will be missing the
+    // generic workspace specification, and will have some
+    // extraneous cargo-specific stuff that we don't want.
+    if is_migrating {
+        // Remove Cargo-specific workspace
+        workspace_toml.remove("workspace");
+
+        // Write generic workspace config
+        let mut table = toml_edit::table();
+        if let Some(t) = table.as_table_mut() {
+            let mut array = toml_edit::Array::new();
+            array.push("cargo:.");
+            t["members"] = toml_edit::value(array);
+            t.set_position(0);
```

Comment:
what on earth does this do

Reply:
OHHH

ok yeah maybe a comment to explain this is adding the Cargo.toml as a member of the workspace haha (also i wonder if `./` will be more legible?